src/main/java/ch/bfh/bti7081/Presenter/MessagePresenter.java

```diff
@@ -0,0 +1,27 @@
+package ch.bfh.bti7081.Presenter;
+
+import model.entities.Message;
+import model.message.MessageQuerier;
+import org.bson.types.ObjectId;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class MessagePresenter {
+
+    private MessageQuerier messageQuerier;
+
+    public MessagePresenter(){
+        messageQuerier = new MessageQuerier();
+    }
+
+    public List<Message> getMessagesByReportId(ObjectId reportId){
+        // TODO: Sort them by sent date eventually
+        return messageQuerier.getAll()
+                .stream()
```

Comment:
presenter should not contain filtering. Please move to querier

Reply:
will do